quisby/pricing/cloud_pricing.py

```diff
@@ -73,34 +73,21 @@ def get_aws_instance_info(instance_name, region):
 
     returns: integer pricing in USD
     """
-    pricing = boto3.client("pricing")
-
-    OPERATING_SYSTEM = "Linux"
-    OPERATION = "RunInstances"
-    TENANCY = "Shared"
-    PRE_INSTALLED_SW = "NA"
-    CAPACITY_STATUS = "UnusedCapacityReservation"
+    region="us-east-1"
```

Comment:
This is fine as a default, but we need to be able to override this with a different region.